org.serioussam.SeriousSamClassic-VK.yaml

```diff
@@ -0,0 +1,118 @@
+app-id: org.serioussam.SeriousSamClassic-VK
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+runtime-version: '23.08'
+command: serioussam.sh
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+
+cleanup:
+  - /include
+  - /lib/*.a
+  - /lib/*.la
+
+modules:
+
+  - name: serioussam
+    buildsystem: cmake-ninja
+    no-make-install: true
+    config-opts:
+      - -DCMAKE_INSTALL_PREFIX:PATH="/usr"
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+    post-install:
+      - install -Dm 755 SamTFE/Sources/serioussam /app/bin/serioussam
```

Comment:
there is no install rules in these cmakefiles?

Reply:
or is it because you imporperly set the prefix that the install fail and you chose to skip it.

Reply:
There are rules, but they are tied to the /usr directory.
So I decided to use manual installation.